src/Sylius/Bundle/ShopBundle/Resources/views/Checkout/address.html.twig

```diff
@@ -2,40 +2,21 @@
 
 {% form_theme form '@SyliusUi/Form/theme.html.twig' %}
 
-{% block content %}
```

Comment:
yeah, will have time at the end of the week. It is planned not to rename the block at all, i'll revert that change